### ISSUE TITLE
HRTF 3D positional audio support

### ISSUE THREAD

**Issue by jan-kleks** [Nibre/MotherVR#234]
If implemented well, 3D positional audio can GREATLY improve immersion -- imagine this feature combined with the VR... Check out for instance this video with headphones: [System Shock 2 with 3D spatial sound 🎧 (OpenAL Soft HRTF audio)](https://www.youtube.com/watch?v=193xuNu0GqA)

And here's comprehensive [List of video games with binaural audio](https://docs.google.com/document/d/1uAklgDNC_LBOYSBbIzBeso6I4laForovIWiTYx6Ss8k/) (where you can find links to more videos, and to the 3D Game Audio server community)

**Comment by jan-kleks** [comment]
>Alien Isolation already supports this, but microsoft broke directsound3d support ever since windows 7. You can do the following.
Step1: Use DSOAL. This is a wrapper that translates directsound3d calls to the openAL api.

That would be really, really great news, but isn't DirectSound3D support in a 2014 game a bit unusual? Last commercial games which support it seem to be from around 2008 according to [this list](https://docs.google.com/document/d/1SyOw7D0H2ecGxzPsCFEFKwupU8LuazUrlkI-gVTpyhc/).

**Comment by gravitone** [comment]
Yes you are correct, I confused Alien Isolation with an older alien title title. As far as I could find, it alien isolation itself uses the wwise audio engine/api, and can do it's own spatialization. You should probably set the game to output in home theater mode, this should make the game itself output 5.1 or 7.1. You can then use one of the new windows supported dolby access or dts connect apps to do the headphone virtualization. I just did a quick test with isolation, and it seems to work when I enable the dolby connect feature on the rift audio device in the windows sound settings. This integrates properly in the modern windows audio pathway and works with practically every modern game. The DTS and dolby options sound heaps better then the windows sonic for heaphones option which is the only free one out of the three.
